lib/runTestFixtures.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+path = require('path');
```

Comment:
Missing `const`. Add the ESLint rule https://eslint.org/docs/rules/no-implicit-globals to fix this.

Maybe just port the .eslintrc.js from our other projects, as Fergal suggested?

Reply:
So the other projects use babel-eslint, where this project is using CommonJS per the Google doc. I could port the .eslintrc.js from the other projects, but that would also require going in and changing all the CommonJS import / exports